[2024-01-12T10:17:01.261] [ERROR] error - CustomError: Data Tidak ada
    at D:\Dev SIPPP\RebuildAsset\src\controllers\web\trxrequestpemakaian-controller.ts:92:10
    at Generator.next (<anonymous>)
    at fulfilled (D:\Dev SIPPP\RebuildAsset\src\controllers\web\trxrequestpemakaian-controller.ts:5:58)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 499,
  status: 'error'
}
